src/js/components/SharedComponents/ProgressComponent.jsx

```diff
@@ -13,8 +13,11 @@ const propTypes = {
 
 const defaultProps = {
     currentStep: 1,
-    totalSteps: 4,
-    stepNames: ['Name Submission', 'Upload .CSV Files', 'Validate .CSV Data', 'Review & Publish']
+    totalSteps: 5,
+    // stepNames: ['Name Submission', 'Upload .CSV Files', 'Validate .CSV Data', 'Review & Publish']
+    stepNames: ['Upload Files', 'Generate D', 'Cross File', 'Generate EF', 'Review & Publish'],
```

Comment:
The first one should probably be "Validate Data" rather than "Upload Files" since it takes you back to the validation step, not the file upload step.